src/rev_ai/apiclient.py

```diff
@@ -324,6 +325,18 @@ def get_account(self):
         """
         url = urljoin(self.base_url, 'account')
         response = self.session.get(url)
-        response.raise_for_status()
+        self._handle_api_failure(response)
 
         return Account.from_json(response.json())
+
+    def _handle_api_failure(self, response):
+        """Helper function to provide more detailed error messages for the user
+        
+        :param response: response from the HTTP request
+        :raises: HTTPError
+        """
+        try:
+            response.raise_for_status()
+        except Exception as err:
+           err.args = (response.text,)
```

Comment:
Why are we returning the JSON?
```
HTTPError: {"title":"Authorization has been denied for this request.","status":401}
```
doesn't seem better than then vanilla
```
HTTPError: 401 Client Error: Unauthorized for url: https://api.rev.ai/speechtotext/v1/account
```
to me.

Reply:
In this case it's not better than vanilla but if we were to take 409 as an example (we return this code when a customer tries to get the transcript of an in progress job) it would be 
`HTTPError: 409 Client Error: Conflict: https://api.rev.ai/speechtotext/v1/jobs/.../transcript`
which is much worse than 
`HTTPError: {
    "allowed_values": [
        "transcribed"
    ],
    "current_value": "in_progress",
    "type": "https://www.rev.ai/api/v1/errors/invalid-job-state",
    "title": "Job is in invalid state",
    "status": 409,
    "detail": "Job is in progress or failed"
}`

Reply:
There definitely shouldn't be any difference in the amount of information the customer has access to between the api and the sdks. In this case the two options that come to mind are to do this or to return the json object, which I definitely don't think we should do as it breaks the promise of the method